### PR TITLE
Add description of TiCDC alert rules

### PR DESCRIPTION
**Pull request by zhaoxinyu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add description of TiCDC alert rules.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-alert-rules.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+* 报警规则：
+
+    (time() - ticdc_processor_checkpoint_ts / 1000) > 600
+
+* 规则描述：
+
+    cdc processor checkpoint延迟超过10分钟。
+
+* 处理方法：
+
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:26]:
```suggestion

    参考 [`TiCDC 同步任务出现中断`](/ticdc/troubleshoot-ticdc.md#ticdc-同步任务出现中断) 的处理方法。
```

File: ticdc/ticdc-alert-rules.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+#### `cdc_resolvedts_high_delay`
+
+* 报警规则：
+
+    (time() - ticdc_processor_resolved_ts / 1000) > 300
+
+* 规则描述：
+
+    cdc processor resolved ts延迟超过5分钟。
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:36]:
```suggestion
    TiCDC 某一个同步任务的 resolved ts 延迟超过10分钟。
```

File: ticdc/ticdc-alert-rules.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+* 处理方法：
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:38]:
```suggestion
* 处理方法：

    该告警与同步任务中断类似，可参考 [`TiCDC 同步任务出现中断`](/ticdc/troubleshoot-ticdc.md#ticdc-同步任务出现中断) 的处理方法。
```

File: ticdc/ticdc-alert-rules.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+#### `ticdc_processor_exit_with_error_count`
+
+* 报警规则：
+
+    `changes(ticdc_processor_exit_with_error_count[1m]) > 0`
+
+* 规则描述：
+
+    cdc processor异常退出。
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:48]:
```suggestion
    TiCDC 某一个同步任务报错退出。
```

File: ticdc/ticdc-alert-rules.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+
+* 处理方法：
+
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:51]:
```suggestion

    参考 [`TiCDC 同步任务出现中断`](/ticdc/troubleshoot-ticdc.md#ticdc-同步任务出现中断) 的处理方法。
```

File: ticdc/ticdc-alert-rules.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+#### `cdc_multiple_owners`
+
+* 报警规则：
+    
+    `sum(rate(ticdc_owner_ownership_counter[30s])) >= 2`
+
+* 规则描述：
+
+    TiCDC集群有多个owner。
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:65]:
```suggestion
    TiCDC 集群有多个 owner。
```

File: ticdc/ticdc-alert-rules.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+
+* 处理方法：
+
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:68]:
```suggestion

    收集 TiCDC 日志，定位原因。
```

File: ticdc/ticdc-alert-rules.md [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+#### `ticdc_mounter_unmarshal_and_mount_time_more_than_1s`
+
+* 报警规则：
+
+    `histogram_quantile(0.9, rate(ticdc_mounter_unmarshal_and_mount_bucket[1m])) * 1000 > 1000`
+
+* 规则描述：
+
+    cdc_mounter unmarshall跟mount的时间超过1秒。
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:78]:
```suggestion
    TiCDC 某一同步任务解码变更数据的耗时超过 1 秒。
```

File: ticdc/ticdc-alert-rules.md [hunk 9]
```diff
@@ -0,0 +1,184 @@
...
+
+* 处理方法：
+
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:81]:
```suggestion

    收集 TiCDC 日志，定位原因。
```

File: ticdc/ticdc-alert-rules.md [hunk 10]
```diff
@@ -0,0 +1,184 @@
...
+#### `cdc_sink_execute_duration_time_more_than_10s`
+
+* 报警规则：
+
+    `histogram_quantile(0.9, rate(ticdc_sink_txn_exec_duration_bucket[1m])) > 10`
+
+* 规则描述：
+
+    cdc sink操作执行时间超过10s。
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:91]:
```suggestion
    TiCDC 某一同步任务写下游执行时间超过 10 秒。
```

File: ticdc/ticdc-alert-rules.md [hunk 11]
```diff
@@ -0,0 +1,184 @@
...
+
+* 处理方法：
+
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:94]:
```suggestion

    检查下游是否出现问题。
```

File: ticdc/ticdc-alert-rules.md [hunk 12]
```diff
@@ -0,0 +1,184 @@
...
+* 报警规则：
+
+    `changes(ticdc_processor_checkpoint_ts[1m]) < 1`
+
+* 规则描述：
+
+    cdc processor checkpoint时间戳1分钟没有推进。
+
+* 处理方法：
+
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:107]:
```suggestion

    参考 [`TiCDC 同步任务出现中断`](/ticdc/troubleshoot-ticdc.md#ticdc-同步任务出现中断) 的处理方法。
```

File: ticdc/ticdc-alert-rules.md [hunk 13]
```diff
@@ -0,0 +1,184 @@
...
+#### `cdc_processor_checkpoint_tso_no_change_for_1m`
+
+* 报警规则：
+
+    `changes(ticdc_processor_checkpoint_ts[1m]) < 1`
+
+* 规则描述：
+
+    cdc processor checkpoint时间戳1分钟没有推进。
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:104]:
```suggestion
    TiCDC 某一个同步任务进度超过 1 分钟没有推进。
```

File: ticdc/ticdc-alert-rules.md [hunk 14]
```diff
@@ -0,0 +1,184 @@
...
+
+#### `ticdc_puller_entry_sorter_sort_bucket`
+
+* 报警规则：
+
+    `histogram_quantile(0.9, rate(ticdc_puller_entry_sorter_sort_bucket{}[1m])) > 1`
+
+* 规则描述：
+
+    ticdc puller entry sorter排序延迟太高。
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:117]:
```suggestion
    TiCDC puller entry sorter 排序延迟太高。
```

File: ticdc/ticdc-alert-rules.md [hunk 15]
```diff
@@ -0,0 +1,184 @@
...
+
+* 处理方法：
+
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:120]:
```suggestion

    收集 TiCDC 日志，定位原因。
```

File: ticdc/ticdc-alert-rules.md [hunk 16]
```diff
@@ -0,0 +1,184 @@
...
+#### `ticdc_puller_entry_sorter_merge_bucket`
+
+* 报警规则：
+
+    `histogram_quantile(0.9, rate(ticdc_puller_entry_sorter_merge_bucket{}[1m])) > 1`
+
+* 规则描述：
+
+    ticdc puller entry sorter merge延迟太高。
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:130]:
```suggestion
    TiCDC puller entry sorter merge延迟太高。
```

File: ticdc/ticdc-alert-rules.md [hunk 17]
```diff
@@ -0,0 +1,184 @@
...
+
+* 处理方法：
+
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:133]:
```suggestion

    收集 TiCDC 日志，定位原因。
```

File: ticdc/ticdc-alert-rules.md [hunk 18]
```diff
@@ -0,0 +1,184 @@
...
+#### `tikv_cdc_min_resolved_ts_no_change_for_1m`
+
+* 报警规则：
+
+    `changes(tikv_cdc_min_resolved_ts[1m]) < 1 and ON (instance) tikv_cdc_region_resolve_status{status="resolved"} > 0`
+
+* 规则描述：
+
+    tikv cdc最小的resolved ts 1分钟没推进。
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:143]:
```suggestion
    TiKV CDC 模块最小的 resolved ts 1 分钟没推进。
```

File: ticdc/ticdc-alert-rules.md [hunk 19]
```diff
@@ -0,0 +1,184 @@
...
+
+* 处理方法：
+
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:146]:
```suggestion

    收集 TiKV 日志，定位原因。
```

File: ticdc/ticdc-alert-rules.md [hunk 20]
```diff
@@ -0,0 +1,184 @@
...
+#### `tikv_cdc_scan_duration_seconds_more_than_10min`
+
+* 报警规则：
+
+    `histogram_quantile(0.9, rate(tikv_cdc_scan_duration_seconds_bucket{}[1m])) > 600`
+
+* 规则描述：
+
+    tikv cdc扫描耗时超过10分钟。
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:156]:
```suggestion
    TiKV CDC 模块的增量扫描耗时超过10 分钟。
```

File: ticdc/ticdc-alert-rules.md [hunk 21]
```diff
@@ -0,0 +1,184 @@
...
+
+* 处理方法：
+
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:159]:
```suggestion

    收集 TiCDC 监控和 TiKV 日志，定位原因。
```

File: ticdc/ticdc-alert-rules.md [hunk 22]
```diff
@@ -0,0 +1,184 @@
...
+#### `ticdc_sink_mysql_execution_error`
+
+* 报警规则：
+
+    `changes(ticdc_sink_mysql_execution_error[1m]) > 0`
+
+* 规则描述：
+
+    cdc sink到下游mysql时遇到错误。
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:169]:
```suggestion
    TiCDC 某一同步任务写下游 MySQL 时遇到错误。
```

File: ticdc/ticdc-alert-rules.md [hunk 23]
```diff
@@ -0,0 +1,184 @@
...
+
+* 处理方法：
+
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:172]:
```suggestion

    MySQL 报错的情况较多，参考[`TiCDC 常见问题和故障处理`](/ticdc/troubleshoot-ticdc.md)
```

File: ticdc/ticdc-alert-rules.md [hunk 24]
```diff
@@ -0,0 +1,184 @@
...
+#### `ticdc_memory_abnormal`
+
+* 报警规则：
+
+    `go_memstats_heap_alloc_bytes{job="ticdc"} > 1e+10`
+
+* 规则描述：
+
+    TiCDC 堆内存使用量超过10GB。
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:182]:
```suggestion
    TiCDC 堆内存使用量超过 10GB。
```

File: ticdc/ticdc-alert-rules.md [hunk 25]
```diff
@@ -0,0 +1,184 @@
...
+    
+* 处理方法：
```
Comment (overvenus) [inline ticdc/ticdc-alert-rules.md:184]:
```suggestion
* 处理方法：

    收集 TiCDC 监控和 TiCDC 日志，定位原因。
```

**Comment by TomShawn** [comment]
@overvenus Assign 给了自己，是由你提供英文翻译吗？

**Comment by hfxsd** [comment]
LGTM

**Review by overvenus — APPROVED** [review]

**Comment by overvenus** [comment]
> @overvenus Assign 给了自己，是由你提供英文翻译吗？

不是，麻烦你们帮忙提供下，谢谢！

File: alert-rules.md [hunk 26]
```diff
@@ -786,6 +786,10 @@ aliases: ['/docs-cn/dev/alert-rules/','/docs-cn/dev/reference/alert-rules/']
 
 关于 TiDB Binlog 报警规则的详细描述，参见 [TiDB Binlog 集群监控报警文档](/tidb-binlog/monitor-tidb-binlog-cluster.md#监控报警规则)。
 
+## TiCDC 报警规则
+
+关于TiCDC 报警规则的详细描述，参见 [TiCDC 集群监控报警](/ticdc/ticdc-alert-rules.md)
```
Comment (Liuxiaozhen12) [inline alert-rules.md:791]:
```suggestion
关于TiCDC 报警规则的详细描述，参见 [TiCDC 集群监控报警](/ticdc/ticdc-alert-rules.md)。
```

**Review by TomShawn — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Pull request merged**